recipes/nm.rb

```diff
@@ -29,13 +29,13 @@
 
 
 
-if node['install']['cloud'].casecmp?("gce") == 0
+if node['install']['cloud'].casecmp?("gcp") == 0
```

Comment:
https://github.com/logicalclocks/conda-chef/blob/a136886cb127b68b2d1e2530a900a504b0a5b5fe/metadata.rb#L93 Can you fix also this line, so users know what to set?

Reply:
ok, had done that.